src/main/java/org/onebusaway/alexa/handlers/intent/IntentHandler.java

```diff
@@ -117,6 +117,34 @@ private void personalizationInitialization() {
         }
     }
 
+    /**
+     * Retrieve the ObaUserDataItem from DynamoDB,
+     * this method will replicate ObaData from default(user level) and apply it to person
+     * when user opted in personalization without stop information.
```

Comment:
We should replicate the default user level info to the person level in all cases when bootstrapping the personalization.

For example, the first time I run OBA after personalization is released, if I have the following saved from previous use:
* City = Tampa
* StopID = 6497
* Route Filter = Stop ID 6497 {6, 10, 15}, Stop ID 1234 {9, 8, 9}
* ...

...then when personalization starts, my personalized record should have the identical info:
* City = Tampa
* StopID = 6497
* Route Filter = Stop ID 6497 {6, 10, 15}, Stop ID 1234 {9, 8, 9}
* ...

As a result, the first time I hear "Sean, Route 1 is arriving in 5 minutes" the same city, stop ID, and route filter (and other settings) should immediately be used.

Reply:
Yean, actually right now, we 
1. We try to retrieve the person level information if person's voice was recognized by Alexa
1. If person level information is not available. try to retriever the user level info.
1. if user level info is available, copy it to person level info (by replace the userId to personId and save it to DB)
So default(User) and person will use the same city name and stop number until person explicit ask OBA “change the city and stop number"